README.md

```diff
@@ -4,23 +4,40 @@ This README outlines the details of collaborating on this Ember addon.
 
```

Comment:
Probably should delete this and replace it with a one line description of what this does.